src/app/home/home.page.html

```diff
@@ -47,18 +47,26 @@ <h1>{{banner?.title}}</h1>
       <ion-item lines="none">
         <ion-grid>
           <ion-row>
-            @for (listData of listResData; track listData.id) {
-            @if (listData?.type == 'solutionList'){
-            <ion-col size="6" size-md="3" (click)="navigateTo(listData?.listingData)">
+            @if (listResData[1]?.type === 'solutionList'){
```

Comment:
Why are we taking the index as 1
Is this a generic way?

Reply:
made it in a generic way